### ISSUE TITLE
Tensorboard fails to add summaries without any warning

### ISSUE THREAD

**Issue by chenlheng** [tensorflow/tensorboard#299]
Hey guys I'm learning to use tensorboard and something wired happens. Tensorboard failed to add summary more than 10 times and showed no warning.Here is my code:
```
import tensorflow as tf
import numpy as np

def rmse(y, y_hat):
    """Compute root mean squared error"""
    return tf.sqrt(tf.reduce_mean(tf.square((y - y_hat))))

def forward(x, e):
    """Forward pass for our fuction"""
    # tensorflow has automatic broadcasting 
    # so we do not need to reshape e manually
    return tf.pow(x, e) 

n = 100 # number of examples
learning_rate = 5e-6

x = tf.placeholder(tf.float32)
y = tf.placeholder(tf.float32)

exp = tf.constant(2.0)
exp_hat = tf.Variable(4.0, name='exp_hat')

y_hat = forward(x, exp_hat)

loss = rmse(y, y_hat)
opt = tf.train.GradientDescentOptimizer(learning_rate)

loss_summary = tf.summary.scalar("loss", loss)
exp_summary = tf.summary.scalar("exp", exp_hat)
all_summaries = tf.summary.merge_all()

train_op = opt.minimize(loss) 

x_train = np.random.rand(n) + 10
y_train = x_train ** 2

loss_history = []
exp_history = []

with tf.Session() as sess:
    
    # Initialize all defined variables
    tf.global_variables_initializer().run()
    
    summary_writer = tf.summary.FileWriter('./tensorboard', sess.graph)
    
    for i in range(0, 500):
        print("Iteration %d" % i)
        summaries, curr_loss, curr_exp, _ = sess.run([all_summaries, loss, exp_hat, train_op], feed_dict={x: x_train, y: y_train})
        
        print("loss = %s" % curr_loss)
        print("exp = %s" % curr_exp)
        
        loss_history.append(curr_loss)
        exp_history.append(curr_exp)
        
        summary_writer.add_summary(summaries, i)
```

The code is from https://medium.com/towards-data-science/pytorch-vs-tensorflow-spotting-the-difference-25c75777377b

The result in console is:
`Iteration 8
loss = 2975.89
exp = 3.36951
Iteration 9
loss = 2721.11
exp = 3.33606`
The program stops here and would not go on without any messages. I did the debug and found it stopped at add_summary(). It's wired because even if I change log frequency to adding 1 summary every 10 iterations, it will still stop at the 90th iteration.
Is it a bug or something?

**Comment by chihuahua** [comment]
I can successfully run that code within my python interpreter. 

> ...
> Iteration 490
> loss = 0.0316113
> exp = 2.00012
> Iteration 491
> loss = 0.300696
> exp = 1.99883
> Iteration 492
> loss = 0.0306275
> exp = 2.00012
> Iteration 493
> loss = 0.301677
> exp = 1.99883
> Iteration 494
> loss = 0.0296427
> exp = 2.00011
> Iteration 495
> loss = 0.302658
> exp = 1.99882
> Iteration 496
> loss = 0.0286592
> exp = 2.00011
> Iteration 497
> loss = 0.303641
> exp = 1.99882
> Iteration 498
> loss = 0.0276752
> exp = 2.00011
> Iteration 499
> loss = 0.304621
> exp = 1.99882
> 

I am running TensorFlow 1.3 on the GPU for python 2.7. How about you?

**Comment by chenlheng** [comment]
@chihuahua Thank you for your reply!
I am running TensorFlow 1.2.1 on CPU for python 2.7

**Comment by chihuahua** [comment]
This issue seems more related to TensorFlow. Could you please file this issue under TensorFlow's repo? https://github.com/tensorflow/tensorflow

**Comment by chenlheng** [comment]
@chihuahua  Thank you for your patience and I will file the issue later.
